SPECS/nss/nss.spec

```diff
@@ -163,7 +180,21 @@ popd
 %{_libdir}/libsoftokn3.so
 %{unsupported_tools_directory}/shlibsign
 
+%files tools
+%{_bindir}/certutil
```

Comment:
It looks like these binaries will be double-packaged here and in the main package (due to the wildcard on the main package). Can you please either exclude these binaries from the main package, or change the main package to explicitly enumerate which binaries are packaged? (Personally, my preference is the second solution, but feel free to do what makes sense here)

Reply:
Yep nice catch. Appreciate it